static/extensions/PuzzlingGGG/underspeech.js

```diff
@@ -0,0 +1,196 @@
+class Underspeech {
+  constructor() {
+    this.dialogueContainer = this.createDialogueContainer();
+    const canvas = document.getElementById('sulfurous') || document.querySelector('canvas');
```

Comment:
use `Scratch.vm.renderer.canvas` to get the canvas

Reply:
vm just redirects to runtime, its better to just use whichever you have the most readily available

Reply:
never mind that metjod is even dummer then whay i thought